receiver/mongodbatlasreceiver/alerts.go

```diff
@@ -289,6 +514,120 @@ func payloadToLogs(now time.Time, payload []byte) (plog.Logs, error) {
 	return logs, nil
 }
 
+type cachePutOptions struct {
+	Expires bool          `mapstructure:"expires"`
```

Comment:
Is there ever a case where Expires = false? Can we omit this?

Reply:
Yeah I think something like this is probably more appropriate: 

```go
	ExpiresAt time.Time `mapstructure:"expires_at"`

```